pybossa/api/task.py

```diff
@@ -22,7 +22,7 @@
     * tasks
 
 """
-from flask import abort
+from flask import abort, current_app
```

Comment:
current_app is already imported at line 38

Reply:
Yes, I removed line 38